ur_controllers/package.xml

```diff
@@ -5,12 +5,20 @@
   <version>0.0.0</version>
   <description>Provides controllers that use the speed scaling interface of Universal Robots.</description>
 
+  <author>Marvin Große Besselmann</author>
+  <author>Lovro Ivanov</author>
+
   <maintainer email="grosse@fzi.de">Marvin Große Besselmann</maintainer>
+  <maintainer email="lovro.ivanov@gmail.com">Lovro Ivanov</maintainer>
+  <maintainer email="denis.stogl@picknik.ai">Denis Stogl</maintainer>
+  <maintainer email="zelenak@picknik.ai">Andy Zelenak</maintainer>
 
-  <license>Apache-2.0</license>
+  <license>Apache License 2.0</license>
```

Comment:
`Apache-2.0` is the SPDX license identifier. There were (and are I believe) good reasons to use that.

See: https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/145.

Reply:
Oh, yeah, you are right! I mixed this one. Thanks for pointing out!